prime-router/src/main/kotlin/azure/Event.kt

```diff
@@ -207,10 +207,11 @@ class ProcessEvent(
     }
 
     override fun hashCode(): Int {
+        // vars used in hashCode() must match those in equals()
         return (7 * eventAction.hashCode()) +
             (31 * reportId.hashCode()) +
             (17 * at.hashCode()) +
-            (19 * retryDuration.hashCode())
+            (19 * retryToken.hashCode())
```

Comment:
This just seems like a bug. hashCode() variables should match `equals()` and not contain a constant?

Note, this bug popped out when renaming the constant array `retryDuration` to `retryDurationInMin` ... which makes me wonder if `retryDurationMapInMin` isn't a better name?

Reply:
OK